lib/metasploit/framework/login_scanner/zabbix.rb

```diff
@@ -76,8 +76,8 @@ def send_request(opts)
           res = cli.send_recv(req)
 
           # Found a cookie? Set it. We're going to need it.
-          if res && res.get_cookies =~ /zbx_sessionid=(\w*);/i
-            self.zsession = $1
+          if res && res.get_cookies =~ /(?:zbx_session(?:id)?)=([a-zA-Z0-9]*)((?:%3D){0,2});/i
+            self.zsession = $1 + $2
```

Comment:
If you did capture the cookie name, you could update this to:
```
self.zsession = $1 + '=' + $2
```

and then later on you'd only need to set one cookie because you know the value that's being used:
```
'Cookie'  => self.zsession
```

Reply:
Ooh nice point, I'll see if I can't do this :)

Reply:
@smcintyre-r7 This should now be resolved in https://github.com/rapid7/metasploit-framework/pull/14297/commits/c62001c21059b3283acab75af6d45a0cc0870eb3